doc/cookbook/source/examples/distance/euclidean.rst

```diff
@@ -0,0 +1,43 @@
+==================
+Euclidean Distance
+==================
+
+The Euclidean distance for real valued features is the square root of the sum of squared disparity between the corresponding feature dimensions of two data points.
+
+.. math::
+
+    d({\bf x},{\bf x'})= \sqrt{\sum_{i=0}^{n}|{\bf x_i}-{\bf x'_i}|^2}
+
+where :math:`\bf x` and :math:`\bf x'` are :math:`n` dimensional feature vectors.
+
+-------
+Example
+-------
+
+Imagine we have files with training and test data. We create CDenseFeatures (here 64 bit floats aka RealFeatures) as
```

Comment:
Can we not call these training and test features? But just features_a, and features_b?